packages/ibm-products/src/components/Datagrid/Datagrid.test.js

```diff
@@ -907,6 +915,28 @@ describe(componentName, () => {
     ).toEqual(10);
   });
 
+  it('renders a basic table and resizes column', async () => {
```

Comment:
`async` is specified, but there's no follow up `await/waitfor` in this test.

Reply:
Fixed